src/medCore/database/medDatabaseController.cpp

```diff
@@ -587,13 +592,6 @@ bool medDatabaseController::moveDatabase( QString newLocation)
             qDebug() << "Restarting connection...";
             this->createConnection();
         }
-
-        // now delete the old archive
-        if(medStorage::removeDir(oldLocation))
```

Comment:
OK I see your point.
But can we have old db if we cannot create a new one without deleting an old one?
(In my case, I got old db because they were created when compiling medInria, MUSIC, etc.)